src/mobile/src/components/Post/Posts.jsx

```diff
@@ -0,0 +1,53 @@
+import Constants from 'expo-constants';
+import { FlatList, View, Button, StyleSheet } from 'react-native';
+import useSWRInfinite from 'swr/infinite';
+import Post from './Post';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+  },
+  divider: {
+    backgroundColor: '#9f9f9f',
+    height: 1,
+    marginHorizontal: 'auto',
+    marginVertical: '1rem',
+    width: '80%',
+  },
+});
+
+const POST_URL = 'https://dev.api.telescope.cdot.systems/v1/posts'; //Constants.manifest.extra.postsURL;
+
+function Posts() {
+  const {
+    data: pages,
+    error,
+    setSize,
+    size,
+  } = useSWRInfinite((index) => `${POST_URL}?page=${index + 1}&per_page=2`);
+
+  if (error || !pages || !pages.length) {
+    return null;
+  }
+
+  const renderPost = ({ item }) =>
+    item.map((post) => (
+      <View key={post.id}>
+        <Post postURL={post.url} />
+        <View style={styles.divider} />
+      </View>
+    ));
+
+  return (
+    <View style={styles.container}>
+      <FlatList
+        data={pages}
+        renderItem={renderPost}
+        keyExtractor={(item, index) => index.toString()}
```

Comment:
Why don't we use `keyExtractor={(item, index) => item.id}`?

So, I read https://reactnative.dev/docs/flatlist#example, it says that 

> `keyExtractor` tells the list to use the ids for the react keys instead of the default key property.

We have the `key` property here 
```jsx
      <View key={post.id}>
        <Post postURL={post.url} />
        <View style={styles.divider} />
      </View>
```

What if we remove the `key` property of `View` and change `keyExtractor` to use `item.id`, would the `key` be the `item.id`?

Reply:
The reason for it cause we have a nested object returned by the `posts` service so it act a bit different then default behavior.